eras/allegra/impl/src/Cardano/Ledger/Allegra/Rules/Utxo.hs

```diff
@@ -212,10 +211,8 @@ utxoTransition = do
   {- txsize tx ≤ maxTxSize pp -}
   runTest $ Shelley.validateMaxTxSizeUTxO pp tx
 
-  let refunded = getTotalRefundsTxBody pp dpstate txb
-  let depositChange = getTotalDepositsTxBody pp dpstate txb Val.<-> refunded
-  tellEvent $ TotalDeposits (hashAnnotated txb) depositChange
-  pure $! Shelley.updateUTxOState pp u txb depositChange ppup'
+  Shelley.updateUTxOState pp utxos txBody certState ppup' $
+    tellEvent . TotalDeposits (hashAnnotated txBody)
```

Comment:
So I can't figure out where the computation of refunds and deposits happens in the new code.

Reply:
In the `updateUTxOState`, which I see that you've discovered it a bit later